network/linphone-desktop/linphone-desktop.SlackBuild

```diff
@@ -0,0 +1,401 @@
+                 mswebrtc \
+                 ortp \
+                 postquantumcryptoengine\
+                 oboe \
+                 webrtc)
+
+
+for i in "${!modulepaths[@]}" ; do
+  echo "${!modulepaths[@]}"
+  tar xvf "$CWD"/"${packagenames[i]}"*
+  rmdir "${modulepaths[i]}"
+  ln -s "$(pwd)"/"${packagenames[i]}"* "${modulepaths[i]}"
+done
+git init >> /dev/null
+git config user.email "dummy@example.test"
+git config user.name "Example"
+git add README.md
+git commit -m "$PKGVERSION" > /dev/null
+git tag -m "$PKGVERSION" "$PKGVERSION"
+echo ""
+
+
+chown -R root:root .
+find -L . \
+ \( -perm 777 -o -perm 775 -o -perm 750 -o -perm 711 -o -perm 555 \
+  -o -perm 511 \) -exec chmod 755 {} + -o \
+ \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
+  -o -perm 440 -o -perm 400 \) -exec chmod 644 {} + || true
+
+my_parallel="${MAKEFLAGS##-j}"
+my_parallel="${my_parallel:-1}"
+
+
+sed -i 's/-Wunused-parameter//g' linphone-sdk/liblinphone/CMakeLists.txt || exit 200
+sed -i '/bc_apply_compile_flags/d' linphone-sdk/liblinphone/console/CMakeLists.txt || exit 200
+
+sed -i 's/cmake_minimum_required(VERSION 3.22)/cmake_minimum_required(VERSION 3.30.5)/g' CMakeLists.txt
+
+sed -i 's/add_dependencies(sdk opencore-amr)/add_dependencies(sdk opencore-amrwb opencore-amrnb)/g' linphone-sdk/cmake/ExternalDependencies.cmake
+sed -i 's/add_dependencies(sdk vo_amrwbenc)/add_dependencies(sdk vo-amrwbenc)/g' linphone-sdk/cmake/ExternalDependencies.cmake
+
+
+
+mkdir -p build
+cd build
+  cmake -G Ninja \
+    -DCMAKE_C_FLAGS:STRING="$SLKCFLAGS" \
+    -DCMAKE_CXX_FLAGS:STRING="$SLKCFLAGS -Wno-error=unused-parameter" \
+    -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+    -DLIB_SUFFIX=${LIBDIRSUFFIX} \
+    -DMAN_INSTALL_DIR=/usr/man \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DENABLE_TOOLS=ON \
+    -DENABLE_PQCRYPTO=ON \
+    -DENABLE_SOCI_MYSQL=ON \
+    -DENABLE_MDNS=ON \
+    -DENABLE_AMRNB=ON \
+    -DENABLE_AMRWB=ON \
+    -DENABLE_CODEC2=ON \
+    -DENABLE_G726=ON -DENABLE_G729=ON -DENABLE_G729B_CNG=ON \
+    -DENABLE_ILBC=ON -DENABLE_ISAC=ON \
+    -DENABLE_SILK=ON \
+    -DENABLE_WEBRTC_VAD=ON -DENABLE_X264=ON \
+    -DENABLE_QRCODE=ON \
+    -DENABLE_RTP_MAP_ALWAYS_IN_SDP=ON \
+    -DENABLE_FFMPEG=ON \
+    -DENABLE_NON_FREE_FEATURES=ON \
+    -DENABLE_OPENH264=ON \
+    -DENABLE_H263=ON -DENABLE_H263P=ON \
+    -DDNSSD_ROOT=/usr/include/avahi-compat-libdns_sd/ \
+    -DENABLE_MDNS=ON \
+    -DENABLE_NLS=ON \
+    -DENABLE_STRICT=NO \
+    -DENABLE_GTK_UI=ON \
+    -DENABLE_DAEMON=ON \
+    -DENABLE_GPL_THIRD_PARTIES=ON \
+    -DENABLE_LIME_X3DH=ON \
+    -DENABLE_CONSOLE_UI=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+  cmake --build . --parallel "$my_parallel"
+  cmake --install . --prefix $PKG/opt/linphone-desktop
+  cd ..
+
+  chmod +x $PKG/opt/linphone-desktop/lib64/*.so*
+
+  (
+    mkdir -p $PKG/usr/bin
+    cd $PKG/usr/bin
+    ln -sr ../../opt/$PRGNAM/bin/linphone $PRGNAM
+  )
+
+rm -f $PKG/{,usr/}lib${LIBDIRSUFFIX}/*.la
```

Comment:
This uses LIBDIRSUFFIX but line 378 hard-codes 64. But DOWNLOAD says 32bit is unsupported. Would be better to be consistent here I think